cmd/setup.go

```diff
@@ -110,8 +110,12 @@ func runSetupError(cmd *cobra.Command, args []string) error {
 	pa := patterns.New(logger)
 	w := wait.New(logger, d, pa)
 	pCfg := &project.Config{
-		Name: projectName,
-		Tag:  projectTag,
+		Name:       projectName,
+		K8sContext: "minikube",
+		Tag:        projectTag,
+	}
+	if existingCluster {
+		pCfg.K8sContext = "giantswarm-e2e"
 	}
```

Comment:
What does `existingCluster` mean? When is it set?

Reply:
its flag, its set when running `e2e-harness setup` command.
Its configuration where e2e is using remote cluster but it won't create it as it will use an existing cluster.

Reply:
BTW to avoid confusion I think it's better to determine value before initializing the struct:

```
k8sContext := "minikube"
if existingCluster {
	k8sContext = "giantswarm-e2e"
}
pCfg := &project.Config{
	Name: projectName,
	Tag:  projectTag,
	Name:       projectName,
	K8sContext: k8sContext,
	Tag:        projectTag,
}
```

Reply:
That would probably complicate things as when running teardown steps it already expects that correct context is used. Lots of stuff would have to be rewritten. Maybe that is out of scope for this PR?

Reply:
fixed